user-interface/6001-SORD-submit_orders.md

```diff
@@ -0,0 +1,139 @@
+# Submit order
+As a user I want change my exposure on a market (e.g. open a position, increase or decrease my open volume), I want to submit an order with instructions for how my order should be executed so I have some control over the price that I get, as well as if when/my order should stay on the book. See [specs about orders](../protocol#orders) for more info.
+
+When populating a deal ticket I...
+
+- **must** see/select the [Market](./7001-DATA-data_display.md#market) I am submitting the order for <a name="6001-SORD-001" href="#6001-SORD-001">6001-SORD-001</a>
+  - **must** see the current market trading mode (Continuous, Auction etc) <a name="6001-SORD-002" href="#6001-SORD-002">6001-SORD-002</a>
+
+- If I have a 0 total balance of the settlement asset: **must** be warned that I have insufficient collateral (but also allow you to populate ticket because I might want to try before I deposit) <a name="6001-SORD-003" href="#6001-SORD-003">6001-SORD-003</a>
+  - **should** see a link to deposit the required collateral <a name="6001-SORD-050" href="#6001-SORD-050">6001-SORD-050</a>
+
+- **must** select a side/direction e.g. long/short (note: some implementations may do this with two different submit buttons long/short rather than a toggle) <a name="6001-SORD-004" href="#6001-SORD-004">6001-SORD-004</a>
+
+- **must** be able to select the [order type](../protocol/0014-ORDT-order_types.md) that I wish to submit <a name="6001-SORD-005" href="#6001-SORD-005">6001-SORD-005</a>
+  - **must** see limit order <a name="6001-SORD-006" href="#6001-SORD-006">6001-SORD-006</a>
+  - **must** see market order <a name="6001-SORD-007" href="#6001-SORD-007">6001-SORD-007</a>
+  - **should** see pegged order <!-- <a name="6001-SORD-008" href="#6001-SORD-008">6001-SORD-008</a> -->
+  - **should** see liquidity provision <!-- <a name="6001-SORD-009" href="#6001-SORD-009">6001-SORD-009</a> -->
+
+## Order size
+
+...need to select a size, when selecting a size for my order, I...
+
+- **must** input an order [size](7001-DATA-data_display.md#size) (aka amount or contracts) <a name="6001-SORD-010" href="#6001-SORD-010">6001-SORD-010</a>
+  - **should** have the previous value pre-populated (last submitted or last changed) <a name="6001-SORD-012" href="#6001-SORD-012">6001-SORD-012</a>
+  - **should** be able to hit up/down on the keyboard to increase the size by the market's min-contract size <a name="6001-SORD-013" href="#6001-SORD-013">6001-SORD-013</a>
```

Comment:
Should be able to use modifier keys (SHIFT or ALT?) to increase/decrease in larger increments with arrows

Reply:
```suggestion
  - **should** be able to hit up/down on the keyboard to increase the size by the market's min-contract size <a name="6001-SORD-013" href="#6001-SORD-013">6001-SORD-013</a>
    - **should** be able to use modifier keys (SHIFT, ALT etc) to increase/decrease in larger increments with arrows <a name="6001-SORD-xxx" href="#6001-SORD-xxx">6001-SORD-xxx</a>
```